### ISSUE TITLE
Highcharts Editor modal is only available in English

### ISSUE THREAD

**Issue by dan-bowerman** [ramp4-pcar4/storylines-editor#278]
The Highcharts Editor modal is not localized and is displaying in English in the Editor, even in French context.

This will require a custom build of the Highcharts Editor. This will also likely require a bunch of translations.

https://github.com/highcharts/highcharts-editor/wiki/Localization

**Comment by RyanCoulsonCA** [comment]
Gave this one a shot but I'm running into a bunch of issues. I'm going to move this back to `To Do`, and I'll put some info below in case someone else wants to give this a shot.

1. I dug through the Highcharts code and replaced a bunch of hardcoded English text with `highed.getLocalizedStr('<translation.string>')` (my code changes are in [this repo](https://github.com/RyanCoulsonCA/highcharts-editor/)). However, there's some English text in the editor that I can't find in the codebase. Not sure where it's coming from, maybe being pulled from elsewhere?
2. The bigger issue is that I can't get the editor to build, both with my changes and without. Below are a few of the errors I'm running into:<br />
**Error 1:** `gulp[17572]: c:\ws\src\node_contextify.cc:696: Assertion 'args[1]->IsString()' failed.` [[GitHub](https://github.com/highcharts/highcharts-editor/issues/214), [StackOverflow](https://stackoverflow.com/questions/50620673/gulp-task-issue-with-nodejs-v10-3-0-src-node-contextify-cc629-assertion-arg)]
- I managed to get this working by running `npm install natives` and `npm audit fix` as recommended in the Stackoverflow thread above.<br />
**Error 2:** `ReferenceError: primordials is not defined` [[StackOverflow](https://stackoverflow.com/questions/55921442/how-to-fix-referenceerror-primordials-is-not-defined-in-node-js)]<br /><br />
- Using the `override` recommendation in the thread fixed this.<br />
**Error 3:** Missing file `highed.editor.config.json`. I created the file and added `{}` to it to get past this issue. Not sure if this file is supposed to be included in the repo or whether what I did is what we're supposed to do.


After fixing these 3 errors the build ran successfully, however when I moved the files in to `Storylines` the chart editor was totally broken and wouldn't work. I've attempted building with 3 different node versions, v6.17.1, v11.0.0, and v18.3.0.
